src/main/java/core/basesyntax/strategy/operation/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy.operation;
+
+public interface OperationHandler {
+    void operationType(String fruit, int amount);
+}
```

Comment:
Consider renaming the 'OperationHandler' interface to something more descriptive, such as 'FruitOperationHandler', to clarify its role in handling fruit operations.